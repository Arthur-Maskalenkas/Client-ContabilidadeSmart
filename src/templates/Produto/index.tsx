import Base from 'templates/Base'
import * as S from './styles'

export type ProdutoTemplateProps = {
  descricaoProduto: string
}

const Produto = ({ descricaoProduto }: ProdutoTemplateProps) => (
  <S.Wrapper>
    <Base>
      <h1>Produto</h1>
    </Base>
  </S.Wrapper>
)

export default Produto
