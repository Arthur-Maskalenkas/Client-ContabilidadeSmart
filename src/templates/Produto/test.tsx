import { render, screen } from '@testing-library/react'
import { renderWithTheme } from 'utils/tests/helpers'

import Produto from '.'

describe('<Produto />', () => {
  it('should render the heading', () => {
    const { container } = renderWithTheme(<Produto />)

    expect(
      screen.getByRole('heading', { name: /Produto/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
