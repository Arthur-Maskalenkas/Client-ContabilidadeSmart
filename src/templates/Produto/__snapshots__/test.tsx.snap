// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Produto /> should render the heading 1`] = `
.c1 {
  width: 100%;
  max-width: 130rem;
  margin-left: auto;
  margin-right: auto;
  padding-left: calc(3.2rem / 2);
  padding-right: calc(3.2rem / 2);
}

.c35 {
  color: #050706;
  font-size: 1.6rem;
}

.c35::after {
  width: 3rem;
}

.c37 {
  color: #050706;
}

.c31 .c34 {
  text-transform: uppercase;
}

.c32 {
  display: grid;
  grid-template-columns: repeat(2,1fr);
  gap: 3.2rem;
  margin-top: 3.2rem;
}

.c33 a,
.c33 span {
  display: block;
  color: #8F8F8F;
  -webkit-text-decoration: none;
  text-decoration: none;
  margin-bottom: 0.8rem;
  font-size: 1.4rem;
}

.c33 a:hover {
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

.c38 {
  color: #8F8F8F;
  font-size: 1.2rem;
  margin-top: 4.0rem;
  margin-bottom: 3.2rem;
  text-align: center;
}

.c36 {
  margin-bottom: 2.4rem;
}

.c5 {
  display: inline-block;
  vertical-align: middle;
  overflow: hidden;
}

.c14 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  color: #FAFAFA;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  background: #F51700;
  font-family: Poppins,-apple-system,BlinkMacSystemFont,'Segoe UI',Roboto,Oxygen,Ubuntu,Cantarell,'Open Sans','Helvetica Neue',sans-serif;
  border: 0;
  cursor: pointer;
  border-radius: 0.4rem;
  padding: 0.8rem;
  -webkit-text-decoration: none;
  text-decoration: none;
  -webkit-transition: 0.3s all;
  transition: 0.3s all;
  height: 4rem;
  font-size: 1.4rem;
  padding: 0.8rem 3.2rem;
}

.c14:hover {
  -webkit-transition: 0.3s all;
  transition: 0.3s all;
  background: #c21200;
}

.c18 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  color: #FAFAFA;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  background: #F51700;
  font-family: Poppins,-apple-system,BlinkMacSystemFont,'Segoe UI',Roboto,Oxygen,Ubuntu,Cantarell,'Open Sans','Helvetica Neue',sans-serif;
  border: 0;
  cursor: pointer;
  border-radius: 0.4rem;
  padding: 0.8rem;
  -webkit-text-decoration: none;
  text-decoration: none;
  -webkit-transition: 0.3s all;
  transition: 0.3s all;
  height: 5rem;
  font-size: 1.6rem;
  padding: 0.8rem 4.8rem;
  width: 100%;
}

.c18:hover {
  -webkit-transition: 0.3s all;
  transition: 0.3s all;
  background: #c21200;
}

.c3 {
  display: none;
}

.c7 {
  display: none;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  padding: 2.4rem 0;
  position: relative;
  z-index: 20;
}

.c6 a {
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c6 p {
  color: #FAFAFA;
  -webkit-letter-spacing: 1rem;
  -moz-letter-spacing: 1rem;
  -ms-letter-spacing: 1rem;
  letter-spacing: 1rem;
  background-color: red;
  padding: 0.2rem;
  border-radius: 0.4rem;
}

.c4 {
  color: #FAFAFA;
  cursor: pointer;
  width: 2.4rem;
  height: 2.4rem;
  -webkit-transition: 0.3s all;
  transition: 0.3s all;
}

.c4:hover {
  -webkit-transition: 0.3s all;
  transition: 0.3s all;
  background: red;
  border-radius: 0.4rem;
}

.c12 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
  -webkit-box-pack: end;
  -webkit-justify-content: flex-end;
  -ms-flex-pack: end;
  justify-content: flex-end;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c12 > div {
  margin-left: 1.6rem;
}

.c9 {
  margin-left: 2.4rem;
}

.c11 {
  cursor: pointer;
  position: relative;
  color: #FAFAFA;
  font-size: 1.6rem;
  margin: 0.3rem 2.4rem 0;
  -webkit-text-decoration: none;
  text-decoration: none;
  text-align: center;
}

.c11:hover::after {
  content: '';
  position: absolute;
  display: block;
  height: 0.3rem;
  background-color: #F51700;
  -webkit-animation: hoverAnimation 0.2s forwards;
  animation: hoverAnimation 0.2s forwards;
}

.c15 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  background: #FAFAFA;
  position: fixed;
  z-index: 20;
  top: 0;
  bottom: 0;
  left: 0;
  right: 0;
  height: 100vh;
  overflow: hidden;
  -webkit-transition: opacity 0.3s ease-in-out;
  transition: opacity 0.3s ease-in-out;
  opacity: 0;
  pointer-events: none;
}

.c15 > svg {
  position: absolute;
  top: 0;
  right: 0;
  margin: 1.6rem;
  cursor: pointer;
  width: 2.4rem;
  height: 2.4rem;
}

.c15 .c8 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c15 .c10 {
  color: #050706;
  font-weight: 600;
  font-size: 2.0rem;
  margin-bottom: 2.4rem;
  -webkit-transform: translateY(3rem);
  -ms-transform: translateY(3rem);
  transform: translateY(3rem);
  -webkit-transition: -webkit-transform 0.3s ease-in-out;
  -webkit-transition: transform 0.3s ease-in-out;
  transition: transform 0.3s ease-in-out;
}

.c15 .c16 {
  -webkit-transform: translateY(3rem);
  -ms-transform: translateY(3rem);
  transform: translateY(3rem);
  -webkit-transition: -webkit-transform 0.3s ease-in-out;
  -webkit-transition: transform 0.3s ease-in-out;
  transition: transform 0.3s ease-in-out;
}

.c17 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  padding: 0 4.8rem 4.8rem;
}

.c17 > span {
  display: block;
  margin: 0.8rem 0;
  font-size: 1.2rem;
}

.c19 {
  -webkit-text-decoration: none;
  text-decoration: none;
  color: #F51700;
  border-bottom: 0.2rem solid #F51700;
}

.c26 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  background: #EAEAEA;
  border-radius: 0.2rem;
  padding: 0 1.6rem;
  border: 0.2rem solid;
  border-color: #EAEAEA;
}

.c26:focus-within {
  box-shadow: 0 0 0.5rem #F51700;
}

.c27 {
  color: #050706;
  font-family: Poppins,-apple-system,BlinkMacSystemFont,'Segoe UI',Roboto,Oxygen,Ubuntu,Cantarell,'Open Sans','Helvetica Neue',sans-serif;
  font-size: 1.6rem;
  padding: 0.8rem 0;
  padding-left: 1.6rem;
  background: transparent;
  border: 0;
  outline: none;
  width: 100%;
}

.c24 {
  font-size: 1.4rem;
  color: #050706;
  cursor: pointer;
}

.c20 {
  position: fixed;
  top: 50%;
  left: 50%;
  -webkit-transform: translate(-50%,-50%);
  -ms-transform: translate(-50%,-50%);
  transform: translate(-50%,-50%);
  max-width: 30rem;
  opacity: 0.8;
  border-radius: 0.4rem;
  background-color: #32A09A;
  padding: 4.8rem;
  padding-top: 3.2rem;
  z-index: 50;
  -webkit-transition: opacity 0.3s ease-in-out;
  transition: opacity 0.3s ease-in-out;
  opacity: 0;
  pointer-events: none;
}

.c20 > svg {
  position: absolute;
  top: 0;
  right: 0;
  margin: 1.6rem;
  cursor: pointer;
  width: 2.4rem;
  height: 2.4rem;
}

.c22 input::-webkit-input-placeholder {
  color: #0e2b2a;
}

.c22 input::-moz-placeholder {
  color: #0e2b2a;
}

.c22 input:-ms-input-placeholder {
  color: #0e2b2a;
}

.c22 input::placeholder {
  color: #0e2b2a;
}

.c22 .c25 {
  background-color: #80CFCF;
  margin-top: 0.8rem;
  -webkit-transition: 0.8s all;
  transition: 0.8s all;
}

.c22 .c25:focus-within {
  box-shadow: 0 0.2rem 1rem #050706;
  background-color: #40a9a9;
}

.c22 .c25:hover {
  -webkit-transition: 0.8s all;
  transition: 0.8s all;
  background-color: #40a9a9;
}

.c22 .c23 {
  margin-top: 0.8rem;
}

.c28 {
  margin-top: 3.2rem;
  -webkit-transition: 0.8s all;
  transition: 0.8s all;
}

.c28 .c13 {
  background-color: #267975;
}

.c28 .c13:hover {
  -webkit-transition: 0.8s all;
  transition: 0.8s all;
  background-color: #0e2b2a;
}

.c21 {
  text-align: center;
  margin-bottom: 2.4rem;
  color: #050706;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  height: 100vh;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

.c29 {
  margin-top: 4.0rem;
  -webkit-flex: 1 0;
  -ms-flex: 1 0;
  flex: 1 0;
}

.c30 {
  margin-top: 4.0rem;
  padding-bottom: 1.6rem;
  padding-top: 5.6rem;
  background-color: #FAFAFA;
  -webkit-clip-path: polygon(0 5%,100% 0%,100% 100%,0 100%);
  clip-path: polygon(0 5%,100% 0%,100% 100%,0 100%);
}

@media (min-width:768px) {
  .c32 {
    grid-template-columns: repeat(4,1fr);
  }
}

@media (max-width:768px) {
  .c3 {
    display: block;
  }
}

@media (min-width:768px) {
  .c7 {
    display: block;
  }
}

@media (max-width:768px) {
  .c6 {
    position: absolute;
    left: 50%;
    -webkit-transform: translateX(-50%);
    -ms-transform: translateX(-50%);
    transform: translateX(-50%);
  }
}

@media (min-width:768px) {
  .c30 {
    padding-top: calc(5.6rem * 2);
    -webkit-clip-path: polygon(0 15%,100% 0%,100% 100%,0 100%);
    clip-path: polygon(0 15%,100% 0%,100% 100%,0 100%);
  }
}

<main
  class=""
>
  <div
    class="c0"
  >
    <div
      class="c1"
    >
      <menu
        class="c2"
      >
        <div
          class="c3"
        >
          <div
            class="c4"
          >
            <svg
              aria-hidden="true"
              aria-label="Abrir menu"
              class="c5"
              fill="currentColor"
              focusable="false"
              viewBox="0 0 24 24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M0 0h24v24H0z"
                fill="none"
              />
              <path
                d="M3 4h18v2H3V4zm0 7h12v2H3v-2zm0 7h18v2H3v-2z"
              />
            </svg>
          </div>
        </div>
        <div
          class="c6"
        >
          <a
            href="/"
          >
            <p>
              MM
            </p>
          </a>
        </div>
        <div
          class="c7"
        >
          <div
            class="c8 c9"
          >
            <a
              class="c10 c11"
              href="/"
            >
              Inicio
            </a>
            <a
              class="c10 c11"
              href="#"
            >
              Sobre
            </a>
            <a
              class="c10 c11"
              href="#"
            >
              Ver noticias
            </a>
          </div>
        </div>
        <div
          class="c12"
        >
          <div
            class="c7"
          >
            <a
              class="c13 c14"
              href="/sign-in"
            >
              <span>
                Criar conta
              </span>
            </a>
          </div>
        </div>
        <nav
          aria-hidden="true"
          class="c15"
        >
          <svg
            aria-hidden="true"
            aria-label="Fechar menu"
            class="c5"
            fill="currentColor"
            focusable="false"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M0 0h24v24H0V0z"
              fill="none"
            />
            <path
              d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41z"
            />
          </svg>
          <div
            class="c8 c9"
          >
            <a
              class="c10 c11"
              href="/"
            >
              Inicio
            </a>
            <a
              class="c10 c11"
              href="#"
            >
              Sobre
            </a>
            <a
              class="c10 c11"
              href="#"
            >
              Ver noticias
            </a>
          </div>
          <div
            class="c16 c17"
          >
            <a
              class="c13 c18"
              href="/sign-in"
            >
              <span>
                Entrar
              </span>
            </a>
            <span>
              or
            </span>
            <a
              class="c19"
              href="/sign-up"
              title="Sign Up"
            >
              Criar conta
            </a>
          </div>
        </nav>
        <form
          aria-hidden="true"
          aria-label="modal de criar noticia"
          class="c20"
          role="form"
        >
          <svg
            aria-hidden="true"
            aria-label="Fechar NoticiaForm"
            class="c5"
            fill="currentColor"
            focusable="false"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M0 0h24v24H0V0z"
              fill="none"
            />
            <path
              d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12 19 6.41z"
            />
          </svg>
          <h2
            class="c21"
          >
            Criar noticia
          </h2>
          <div
            class="c22"
          >
            <div
              class="c23 "
            >
              <label
                class="c24"
                for="titulo"
              >
                titulo
              </label>
              <div
                class="c25 c26"
              >
                <input
                  class="c27"
                  id="titulo"
                  placeholder="Titulo"
                  type="text"
                  value=""
                />
              </div>
            </div>
            <div
              class="c23 "
            >
              <label
                class="c24"
                for="detalhes"
              >
                descrição
              </label>
              <div
                class="c25 c26"
              >
                <input
                  class="c27"
                  id="detalhes"
                  placeholder="Descrição"
                  type="text"
                  value=""
                />
              </div>
            </div>
          </div>
          <div
            class="c28"
          >
            <button
              class="c13 c14"
            >
              <span>
                Criar
              </span>
            </button>
          </div>
        </form>
      </menu>
    </div>
    <div
      class="c29"
    >
      <h1>
        Produto
      </h1>
    </div>
    <section
      class="c30"
    >
      <div
        class="c1"
      >
        <footer
          class="c31"
        >
          <div
            class="c32"
          >
            <div
              aria-labelledby="column-contact"
              class="c33"
            >
              <h2
                class="c34 c35 c36"
                color="black"
              >
                Contact Us
              </h2>
              <a
                href="mailto:sac@wongames.com"
              >
                mm@mm.com
              </a>
            </div>
            <div
              aria-labelledby="social-media"
              class="c33"
            >
              <h2
                class="c34 c37 c36"
                color="black"
              >
                Follow us
              </h2>
              <nav>
                <a
                  href="https://www.instagram.com/mixeria-milionaria"
                  rel="noopenner, noreferrer"
                  target="_blank"
                >
                  Instagram
                </a>
                <a
                  href="https://www.twitter.com/mixeria-milionaria"
                  rel="noopenner, noreferrer"
                  target="_blank"
                >
                  Twitter
                </a>
                <a
                  href="https://www.youtube.com/mixeria-milionaria"
                  rel="noopenner, noreferrer"
                  target="_blank"
                >
                  Youtube
                </a>
                <a
                  href="https://www.facebook.com/mixeria-milionaria"
                  rel="noopenner, noreferrer"
                  target="_blank"
                >
                  Facebook
                </a>
              </nav>
            </div>
            <div
              aria-labelledby="footer-resources"
              class="c33"
            >
              <h2
                class="c34 c37 c36"
                color="black"
              >
                Links
              </h2>
              <nav>
                <a
                  href="/"
                >
                  Inicio
                </a>
                <a
                  href="/mixarias"
                >
                  Mixarias
                </a>
                <a
                  href="/buscar"
                >
                  Buscar
                </a>
              </nav>
            </div>
            <div
              aria-labelledby="footer-contact"
              class="c33"
            >
              <h2
                class="c34 c37 c36"
                color="black"
              >
                Location
              </h2>
              <span>
                Lorem ipsum dolor sit.
              </span>
              <span>
                Lorem Ipsum
              </span>
              <span>
                Lorem, ipsum dolor.
              </span>
            </div>
          </div>
          <div
            class="c38"
          >
            Mixaria Milionaria 2021 © All rights reserved.
          </div>
        </footer>
      </div>
    </section>
  </div>
</main>
`;
